src/wallet/wallet.cpp

```diff
@@ -4057,7 +4056,7 @@ std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(const std::string& name,
         }
     }
 
-    uiInterface.InitMessage(_("Loading wallet..."));
+    uiInterface.InitMessage(strprintf(_("[%s] Loading wallet..."), walletFile));
```

Comment:
nit: This looks a bit weird in the logs compared to the other messages:

```
2018-06-21T14:12:32.894599Z init message: [w1] Loading wallet...
```

(ie the `[<wallet_name>]` prefix is not at the start of the log). Maybe just change to:

```
    uiInterface.InitMessage(strprintf(_("Loading wallet %s ..."), walletFile));
```

Reply:
Fixed